wpk/generate_wpk_package.sh

```diff
@@ -9,55 +9,59 @@
 # License (version 2) as published by the FSF - Free Software
 # Foundation.
 
-CURRENT_PATH="$( cd $(dirname $0) ; pwd -P )"
+CURRENT_PATH="$( cd $(dirname ${0}) ; pwd -P )"
 LINUX_BUILDER="unified_linux_wpk_builder"
 LINUX_BUILDER_DOCKERFILE="${CURRENT_PATH}/unified/linux"
 WIN_BUILDER="windows_wpk_builder"
 WIN_BUILDER_DOCKERFILE="${CURRENT_PATH}/windows"
 CHECKSUM="no"
+lsION_PATH="/var"
 
 trap ctrl_c INT
 
 function build_wpk_windows() {
-  local BRANCH="$1"
-  local DESTINATION="$2"
-  local KEYDIR="$3"
-  local CONTAINER_NAME="$4"
-  local JOBS="$5"
-  local PACKAGE_NAME="$6"
-  local OUT_NAME="$7"
-  local CHECKSUM="$8"
-  local CHECKSUMDIR="$9"
+  local BRANCH="${1}"
+  local DESTINATION="${2}"
+  local KEYDIR="${3}"
+  local CONTAINER_NAME="${4}"
+  local JOBS="${5}"
+  local PACKAGE_NAME="${6}"
+  local OUT_NAME="${7}"
+  local CHECKSUM="${8}"
+  local CHECKSUMDIR="${9}"
+  local DEFAULT_INSTALLATION_PATH="${10}"
```

Comment:
Change the name of this variable to `INSTALLATION_PATH`. The default one is `/var/ossec` but if you can change it, it won't be the default anymore.

Reply:
Fixed https://github.com/wazuh/wazuh-packages/pull/408/commits/266a8eff05ec92416c5eee4eb6f8adbfbf3a07ea